husky_components/datagrid/main.js

```diff
@@ -1414,11 +1414,15 @@
 
             /**
              * Updates the selected-elements counter
-             * @param {String|number} number - the number of selected items
              */
-            updateSelectedCounter: function(number) {
+            updateSelectedCounter: function() {
```

Comment:
no its only internal. Its more convenient if i read the length of an array within the method instead of passing the number around